src/hdusd/utils/matlib.py

```diff
@@ -76,10 +77,15 @@ def request_json(url, params, path, cache_check=True):
         with open(path) as json_file:
             return json.load(json_file)
 
-    log("request_json", f"{url=}, {params=}, {path=}")
+    res_json = None
 
-    response = requests.get(url, params=params)
-    res_json = response.json()
+    while res_json is None or ("detail" in res_json and "Request was throttled" in res_json["detail"]):
+        log("request_json", f"{url=}, {params=}, {path=}")
+
+        response = requests.get(url, params=params)
+        res_json = response.json()
+
+        time.sleep(1)
```

Comment:
If a request is throttled, is 1 sec long enough to wait?

Reply:
@radeonprorender looking for an answer here.

Reply:
Yes, we have limit 1000 request per second